examples/specs/scatter_opacity_yfield.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I use two to show opacity with different fields (from x and y) and `scatter_opacity` is using value instead of scale.